pkgs/development/python-modules/dask/default.nix

```diff
@@ -36,17 +36,9 @@ buildPythonPackage rec {
     owner = "dask";
     repo = pname;
     rev = version;
-    hash = "sha256-zHJR2WjHigUMWtRJW25+gk1fKGKedU53BBjwx5zaodA=";
+    hash = "sha256-avyrKBAPyYZBNgItnkNCferqb6+4yeGpBAZhSkL/fFA=";
   };
 
-  patches = [
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Thanks, @SuperSandro2000. I closed this PR because these new versions are already in master.